main.py

```diff
@@ -15,9 +17,38 @@
 #set colors R,G,B code
 BLACK = (0, 0, 0)
 ORANGE = (255,140,0)
+GREEN = (0,225,0)
+GOLD = (255,223,0)
+
+inst1 = "1. There exists a 5x5 grid, one of which holds the key"
+inst2 = "2. You have 5 tries to find the key"
+inst3a = "3. As you go closer to the key, the temperature increases"
+inst3b = "   and as you go away from the key, it decreases"
+inst4 = "4. If you find the key in 5 turns, you win, else you lose"
+
+instructionSet = []
+
+instructionSet.extend([inst1,inst2,inst3a,inst3b,inst4])
+
+#Start Button
+startRect = pygame.Rect((392,509,240,50))
+#Instruction Button
+howtoRect = pygame.Rect((392,584,240,50))
+#Back Button
+backRect = pygame.Rect((392,584,240,50))
+
+#Function for rendering text
+def text_objects(text, font):
+  textSurface = font.render(text, True, BLACK)
+  return textSurface, textSurface.get_rect()
 
 #Find key in 6 turns
 def SEARCH():

```

Comment:
Change this to `search_game()` on my behalf to be a little more with the naming conventions (a reminder to check your other variables)